### PR TITLE
Updated tutorial descriptions

### PR DESCRIPTION
**Pull request by HelioAnalytics**

### CODE REVIEW

File: .DS_Store
Comment (sapols) [inline .DS_Store:1]: Whoops, a .DS_Store made it into this PR. Please delete it?

File: aiml-tutorial/.DS_Store
Comment (sapols) [inline aiml-tutorial/.DS_Store:1]: Another .DS_Store to delete.

**Review by sapols — CHANGES_REQUESTED** [review]

File: .DS_Store
Comment (HelioAnalytics) [reply .DS_Store:1]: Thanks for catching that.  It should be gone now with only the aiml folder needing to be updated.

Comment (HelioAnalytics) [reply .DS_Store:1]: Is there a way to merge only a specific directory and not the whole forked repository?

Comment (sapols) [reply .DS_Store:1]: Not that I know how to do 🤷‍♂️

File: aiml-tutorial/images/.DS_Store
Comment (sapols) [inline aiml-tutorial/images/.DS_Store:1]: There's still a .DS_Store in `/images` here to delete.

**Review by sapols — CHANGES_REQUESTED** [review]

File: aiml-tutorial/images/.DS_Store
Comment (HelioAnalytics) [reply aiml-tutorial/images/.DS_Store:1]: Argh.  Done!

**Review by sapols — APPROVED** [review]

**Pull request merged**